stdlib/2/cStringIO.pyi

```diff
@@ -1,12 +1,15 @@
 # Stubs for cStringIO (Python 2.7)
 # See https://docs.python.org/2/library/stringio.html
 
+from abc import ABCMeta
 from typing import overload, IO, List, Iterable, Iterator, Optional, Union
 from types import TracebackType
 
 # TODO the typing.IO[] generics should be split into input and output.
```

Comment:
This is actually a good idea. Some other languages have `Reader` and `Writer` classes. We can have structure like this:
```python
class Reader(Generic[AnyStr]):
    # basic read methods
class Writer(Generic[AnyStr]):
   # basic write methods
class IO(Reader[AnyStr], Writer[AnyStr]):
    # some additional rarely used methods
```
Do we have an issue to track this?

Reply:
OK, opened https://github.com/python/typing/issues/564